content/en/docs/tutorials/kubernetes-basics/scale/scale-intro.html

```diff
@@ -38,9 +38,7 @@ <h3>Scaling an application</h3>
             <div class="col-md-4">
                 <div class="content__box content__box_lined">
                     <h3>Summary:</h3>
-                    <ul>
                         <li>Scaling a Deployment</li>
```

Comment:
The inner `<li>` now doesn't make sense.

Reply:
@sftim  One more we can do we can only content_box_lined class only instead of 
<div class="content__box_lined">
<h3> Summary::</h3>
<ul>
<li> Scaling a Deployment</li>
</ul> 
It can also work  
It can look somehow this:
![Screenshot from 2024-01-18 11-15-01](https://github.com/kubernetes/website/assets/124760321/aa6bdda9-70be-4cb4-9ae9-e5bed7aa6fb4)

WDYT ? to use will be the best practice.